splash/network_manager.py

```diff
@@ -332,12 +332,9 @@ def _run_webpage_callbacks(self, request, event_name, *args):
                 # should we raise errors here?
                 # https://github.com/scrapinghub/splash/issues/161
                 self.log("error in %s callback" % event_name, min_level=1)
-                # Log the traceback. It requires escaping because
-                # self.log uses str.format.
-                tb = traceback.format_exc().replace('{', '{{').replace('}', '}}')
-                self.log(tb, min_level=1)
+                self.log(traceback.format_exc(), min_level=1, format_msg=False)
```

Comment:
Tracebacks will be much less readable as json; you can make log parsers parse Python stacktraces, but you can't make json stacktraces readable for humans.